factor/telegram/classes/telegram.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+
+namespace factor_telegram;
+
+
+class telegram {
+
+    private $token;
+
+    public function __construct($token) {
+        $this->token = $token;
+    }
+
+    public function send_message($userid, $text) {
+        $path = "https://api.telegram.org/bot".$this->token;
+        file_get_contents($path."/sendmessage?chat_id=".$userid."&text=".$text);
```

Comment:
Please wrap this in some error handling and use the curl library. I would expect the telegram to be pretty snappy so it could have a very short timeout of say 5-10 seconds and not the default which can hang for some time